src/main/scala/com/github/sbt/sbom/BomExtractorParams.scala

```diff
@@ -3,4 +3,4 @@ package com.github.sbt.sbom
 import org.cyclonedx.Version
 import sbt.Configuration
 
-case class BomExtractorParams(schemaVersion: Version, configuration: Configuration)
+final case class BomExtractorParams(schemaVersion: Version, configuration: Configuration)
```

Comment:
Idiomatically all case classes are assumed to be final. Any attempt to extend them is non-idiomatic, and asking for trouble. For sbt plugins I don't think putting this guardrail is not necessary.